packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -137,39 +158,52 @@ class Send extends Component {
     );
   }
 
+  preValidate = values => {
+    const { balance, token } = this.props;
+    const amount = +values.amount;
+
+    if (!amount || isNaN(amount)) {
+      return { amount: 'Please enter a valid amount' };
+    } else if (amount < 0) {
+      return { amount: 'Please enter a positive amount ' };
+    } else if (balance && balance.lt(amount)) {
+      return { amount: `You don't have enough ${token.symbol} balance` };
+    } else if (!values.to || isNaN(values.to)) {
+      // for gas estimation, the receiving address must be set
+      return false;
```

Comment:
We could maybe return `{ to: 'Please enter a valid Ethereum address' }` here, and remove the same check from validateForm()